1-JavaScript/Week3/homework/ex7-mindPrivacy.js

```diff
@@ -30,8 +30,17 @@ const employeeRecords = [
 ];
 
 // ! Function under test
-function filterPrivateData(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function filterPrivateData(employeeRecordsArr) {
+  if (employeeRecordsArr!==undefined){
+    const employeePublicData=[];
+ for (let employeeData of employeeRecordsArr)
+ {
+  let {name,occupation,email}=employeeData;
```

Comment:
```suggestion
  const {name,occupation,email}=employeeData;
```

Try to get in the habit of using `const` whenever data doesn't change